eng/Directory.Build.Common.targets

```diff
@@ -12,6 +12,11 @@
     <Error Condition="'@(RequiredTargetFrameworks)' == ''" Text="List of required target frameworks is empty something must have messed up property RequiredTargetFrameworks[$(RequiredTargetFrameworks)]." />
     <Error Condition="'@(MissingTargetFrameworks)' != ''" Text="Missing required target frameworks '@(MissingTargetFrameworks)'. Please ensure you add those frameworks." />
   </Target>
+  
+  <Target Name="ValidateRunApiCompat" BeforeTargets="Build" Condition="'$(ValidateRunApiCompatSetting)' == 'true'">
```

Comment:
Nit: could we drop the "Setting" and just name the property the same as the target for 1) consistency, and 2) shorter? Should still work, I think. But, again, a nit.